upload manual (#1)
4castRenewables/windstore: upload_data.md

# Upload external project data to 4cast s3 bucket

The manual assumes that 
- Access credentials have been recieved as csv from 4cast including `Access key ID` and `Secret access key`. 
- [Aws CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) is installed on local machine.
  - Confirm via calling `aws --version`.

IEE please upload the files to `path_s3=s3://windstore-weather-data/iee/`.
### Configure AWS CLI 

Use the option `--profile <profile_name>` to set up access keys for the accounts.
See https://docs.aws.amazon.com/cli/latest/userguide/cli-authentication-user.html for more details.

> The profile will be saved in your `~/.aws` folder. In the `credentials` file are the AWS Access Key ID  and AWS Secret Access Key. The rest of the configurations are in the `config` file. 

Reviewer: ```suggestion
The profile will be saved in your `~/.aws` folder in two files `config` and `credentials`. The former contains the profile, the latter the related  AWS Access Key ID and AWS Secret Access Key. `config` is 
```

```
[default]
region = eu-central-1
output = json

[profile <profile_name>]
region = eu-central-1
output = json
```

> When adding this new profile production, make sure to also add new credentials in ~/.aws/credentials. (you can also use the same key ID and secret key for both profiles)

Reviewer: ```suggestion
When adding this new profile `<profile_name>`, make sure to also add new `credentials` in `~/.aws/credentials` (keys may agree).
```
